AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -54,6 +54,8 @@ newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,nordbayern.de,theloca
 !
 ! SECTION: Subscriptions - Regular rules
 !
+venturebeat.com##.post-boilerplate:has(.ICCta)
```

Comment:
Argh( I meant only `venturebeat.com##.post-boilerplate:has(.Boilerplate__newsletter-container)`. The necessary line was not selected.

`venturebeat.com##.post-boilerplate:has(.ICCta)` was in the correct place.